docs/cli-reference/gardenadm/gardenadm_token.md

```diff
@@ -22,8 +22,8 @@ Manage bootstrap and discovery tokens for gardenadm join
 ### SEE ALSO
 
 * [gardenadm](gardenadm.md)	 - gardenadm bootstraps and manages autonomous shoot clusters in the Gardener project.
-* [gardenadm token create](gardenadm_token_create.md)	 - Create a bootstrap token on the server
 * [gardenadm token delete](gardenadm_token_delete.md)	 - Delete a bootstrap token on the server
 * [gardenadm token list](gardenadm_token_list.md)	 - List all bootstrap tokens on the server
+* [gardenadm token create](gardenadm_token_create.md)	 - Create a bootstrap token on the server for joining a node
```

Comment:
Why did you move this command down here? Doesn't it indeed make more sense if listed first? What was your motivation? 😃